Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/FunctionStartupException.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <copyright file="FunctionStartupException.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Testing.AzureFunctions
+{
+    using System;
+
+    /// <inheritdoc />
+    public class FunctionStartupException : Exception
```

Comment:
I'm getting a Roslynator error for this class because it's not properly implementing all exception constructors (https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1194.md). 

We do this quite a lot in our code, so I'd be happy if you wanted to add
`<NoWarn>RCS1194</NoWarn>`
to the project file.

Reply:
Interesting that Roslynator doesn't produce those errors for me in Visual Studio. Are the Analyzers not included, or do I need to add the extension? I've updated the exception definition rather than suppressed the warning for the moment; do you see the same error from `ProcessStartupException` also?

Reply:
Yes, `ProcessStartupException` was giving me exactly the same error. I'm actually a little confused as to why I haven't noticed that before.